### PR TITLE
Dead code cleanup

### PR DESCRIPTION
**Pull request by hainest**
A bit of weekend yak shaving. No urgency at all.

### CODE REVIEW

**Comment by hainest** [comment]
No test regressions.

File: dyninstAPI/src/image.C [hunk 1]
```diff
@@ -1049,11 +1037,7 @@ pdmodule *image::findModule(const string &name, bool wildcard)
       }
    }
 
-   //cerr << " (image::findModule) did not find module, returning NULL" << endl;
    if (found) {
-      // Just-in-time...
-      //if (parseState_ == symtab)
-      //analyzeImage();
       return found;
    }
```
Comment (kupsch) [inline dyninstAPI/src/image.C:1043]: replace this and the `return NULL;` with `return found;`

**Review by kupsch — APPROVED** [review]
one small comment to make code simpler/clearer.

File: dyninstAPI/src/image.C (hunk 1 above)
Comment (hainest) [reply dyninstAPI/src/image.C:1043]: Fixed. Thanks for the careful look!

**Pull request merged**